docs/_snippets/webhook-signature-validation.mdx

```diff
@@ -0,0 +1,14 @@
+```typescript
+
+  const signature = req.headers["x-metriport-signature"];
+  const signatureAsString = // codeToValidateAndConvertToString(signature);
```

Comment:
req.headers["x-metriport-signature"] is of type string | string[] | unknown, so it needs to be converted to a string because the `verifyWebhookSignature` function takes type string